package.json

```diff
@@ -37,7 +37,6 @@
     "url": "https://github.com/react-native-async-storage/async-storage.git"
   },
   "scripts": {
-    "ci": "yarn --pure-lockfile --non-interactive",
```

Comment:
Note that this is default in v3, when it detects that it is running in a CI environment.